docs/ingestion/tasks.md

```diff
@@ -93,6 +93,59 @@ An example output is shown below:
 }
 ```
 
+For compaction tasks alone, there could be multiple `index`, `index_parallel` tasks run based on the split of input
```

Comment:
`Compaction tasks can generate multiple sets of segment output reports based on how the input interval is split. So the report contains mappings from each split to each report.`